.travis.yml

```diff
@@ -4,26 +4,17 @@ language: generic
 compiler:
   - gcc
   - clang
+notifications:
+  email: false
 env:
-  - CC=gcc CXX=g++
-  - CC=clang CXX=clang++
+  global:
+    - ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - CATKIN_CONFIG='--cmake-args -DCMAKE_CXX_FLAGS="-Werror=uninitialized -Werror=return-type -Werror=format"'
```

Comment:
This could even be simplified to `CXXFLAGS="-Werror=uninitialized -Werror=return-type -Werror=format"` (no need for catkin config).